wire/msgcfilter.go

```diff
@@ -13,9 +13,13 @@ import (
 
 const (
 	// MaxCFilterDataSize is the maximum byte size of a committed filter.
-	MaxCFilterDataSize = 262144
+	// The maximum size is currently defined as 256KiB.
+	MaxCFilterDataSize = 256 * 1024
```

Comment:
How did we arrive at this value? 

IIRC, was from going a power of two up from the largest filter we'd seen with current block sizes?

Reply:
That's correct. This *should* be a safe value for now, but we can make sure when we re-run the test vectors.